src/js/structs/Service.js

```diff
@@ -81,12 +86,32 @@ module.exports = class Service extends Item {
     };
   }
 
+  getStatus() {
+    let {tasksRunning} = this.getTasksSummary();
+    let deployments = this.getDeployments();
+
+    if (deployments.length > 0) {
+      return ServiceStatus.DEPLOYING.displayName;
+    }
+
+    if (tasksRunning > 0) {
+      return ServiceStatus.RUNNING.displayName;
+    }
+
+    let instances = this.getInstances();
+    if (instances === 0) {
+      return ServiceStatus.SUSPENDED.displayName;
+    }
+  }
+
   getTasksSummary() {
     return {
       tasksHealthy: this.get('tasksHealthy'),
       tasksRunning: this.get('tasksRunning'),
       tasksStaged: this.get('tasksStaged'),
-      tasksUnhealthy: this.get('tasksUnhealthy')
+      tasksUnhealthy: this.get('tasksUnhealthy'),
+      tasksUnknown: this.get('tasksRunning') -
```

Comment:
What do we need this for? Isn't this just an agregate of the others? Seems weird to introduce